content/docs/getting-started@ko.md

```diff
@@ -0,0 +1,102 @@
+---
+$title@: 시작하기
```

Comment:
Please remove the `@` in the title variable; it's not needed if the text is translated.